schemas/hs00_event_histogram.fbs

```diff
@@ -0,0 +1,26 @@
+
+file_identifier "hs00";
+
+// Meta information for one dimension
+table DimMetaData {
+    histogram_shape: uint;    // Shape of the full histogram for this dimension
```

Comment:
For this field I think it should be "size", not "shape".

Reply:
Do you mean a `errors` array of the same length (and "shape") as the `data` field, or an overall error counter of the same data type as a single element of the `data` array?